### PR TITLE
AE wireframe of overview started. 

### PR DESCRIPTION
**Pull request by alishaedington**
Changes made:
in index, changed title to project catwalk
in app, changed from class component to functional component to utilize hooks
in overview, began wire-framing work and passing props down using context
created context

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
-      </div>
+      <div>Loading...</div>
     );
   }
+  return (
+    <ProductContext.Provider value={{
+      products,
+    }}
+    >
+      <Overview products={products} />
```
Comment (jordanabbasi) [inline client/src/App.jsx]: don't need to pass products as a prop here if it's in the context, right?

File: client/src/components/Overview/Overview.jsx [hunk 2]
```diff
@@ -1 +1,54 @@
...
+    fetchStyles();
+  }, []);
+
+  if (styles.length === 0) {
+    return (
+      <div>Loading...</div>
+    );
+  }
+  return (
+    <ProductContext.Provider value={{
```
Comment (jordanabbasi) [inline client/src/components/Overview/Overview.jsx:18]: It feels confusing to have two different ProductContext.Providers with different shapes.  Should we just have one?

**Review by jordanabbasi — CHANGES_REQUESTED** [review]

File: client/src/components/Overview/Overview.jsx (hunk 2 above)
Comment (alishaedington) [reply client/src/components/Overview/Overview.jsx:18]: I was unsure about that. If we create a new state in a lower file, as I did here in Overview, should we just create a new context to pass stuff down? Or have all state stored in App, and then pass down accordingly? May need to utilize Promises (probably async await) if we choose to do the latter, so we can avoid errors as we try to use data deeper.

File: client/src/App.jsx (hunk 1 above)
Comment (alishaedington) [reply client/src/App.jsx]: I was following an example from Truman, but I just took them out and it seems to be working.

**Review by jordanabbasi — APPROVED** [review]
Looks great!

**Pull request merged**